media/admin/com_virtuemart/models/zasilkovna.php

```diff
@@ -247,16 +251,15 @@ public function isShipmentPaymentRestrictionInstalled()
     public function checkModuleVersion()
     {
         $checkUrl = $this->_zas_url . "api/" . $this->api_key . "/version-check-virtuemart2?my=" . self::VERSION;
-        $data = json_decode($this->fetch($checkUrl));
+        $data = json_decode($this->downloader->fetch($checkUrl), false);
```

Comment:
Downloader má namespace Carrier\Downloader , týká se tedy carrierů.  Není proto vhodné jej použít v této metodě.   V této metodě bych žádné změny nedělal a nechal bych beze změny.  Hlavní důvod je ten, že celá metoda checkModuleVersion() půjde pryč, protože kontrola nové verze modulů se v Joomle dělá jinak - definice update serveru patří do zasilkovna.xml.  Je proto zbytečné a komplikované tuto metodu refaktorovat.

Reply:
tahle metoda se bezne provolava, takze nemuze zustat nefunkcni.

Reply:
Tuto metodu jsem dnes odstranil aby tam zbytečně neobtěžovala a nebylo ji nutné aktualizovat.  Aktualizace volaná v této metodě se již několik let nepoužívá a bude nahrazena aktualizačním serverem Joomly.